From 6a25305dcd4bb2c42a4802b23f045719a9d7d027 Mon Sep 17 00:00:00 2001
From: Al Bailey <Al.Bailey@windriver.com>
Date: Thu, 4 Jan 2018 10:08:36 -0600
Subject: [PATCH] Remove joshuabaird-ipaclient from puppet-nova requirements

We are not using novajoin in our manifests so we do not need to package and include an additional
puppet module not provided by centos
---
 metadata.json | 1 -
 1 file changed, 1 deletion(-)

diff --git a/metadata.json b/metadata.json
index 26a2c3a..e505b71 100644
--- a/metadata.json
+++ b/metadata.json
@@ -10,7 +10,6 @@
   "dependencies": [
     {"name":"puppetlabs/apache","version_requirement":">=1.0.0 <2.0.0"},
     {"name":"duritong/sysctl","version_requirement":">=0.0.1 <1.0.0"},
-    {"name":"joshuabaird/ipaclient","version_requirement":">=2.5.1"},
     {"name":"openstack/cinder","version_requirement":">=11.3.0 <12.0.0"},
     {"name":"openstack/glance","version_requirement":">=11.3.0 <12.0.0"},
     {"name":"puppetlabs/inifile","version_requirement":">=1.0.0 <2.0.0"},
-- 
1.8.3.1

